invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -269,11 +269,26 @@ class RelatedIdentifierSchema(IdentifierSchema):
 
     def __init__(self, **kwargs):
         """Constructor."""
-        super().__init__(allowed_schemes=record_identifiers_schemes, **kwargs)
+        super().__init__(
+            allowed_schemes=record_identifiers_schemes,
+            group_errors=True,
+            **kwargs
+        )
```

Comment:
With https://github.com/inveniosoftware/marshmallow-utils/pull/47#pullrequestreview-716902088 , we don't need it anymore.